docker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM jenkins/jenkins:2.332.2-lts
+ENV JAVA_OPTS -Djenkins.install.runSetupWizard=false
```

Comment:
We do not plan on removing that. Since plugins are pre-baked, we want to skip the set up wizard entirely. Always! Does that makes sense or we still want to pass that from inside EC2?

Reply:
Just adding some useful URL: https://www.jenkins.io/doc/book/managing/system-properties/

`Set to false to skip install wizard. Note that doing so leaves Jenkins unsecured. Development-mode only: Set to true to not skip showing the setup wizard during Jenkins development. This property is only effective the first time you run Jenkins in given JENKINS_HOME.`
This is only effective for the 1st time, since we cache the data, for our scenario we dont need this.

Reply:
Also we can take advantage of docker-compose if we required as 
```
environment:
     -Djenkins.install.runSetupWizard: false
```